From a19b809b7062ceb80abeb5c9c6271d3c0b4cd749 Mon Sep 17 00:00:00 2001
From: CTXz <patrick_pedersen_02032001@yahoo.com>
Date: Fri, 23 Oct 2015 21:33:50 +0200
Subject: [PATCH 1/1] Added refference for added media defs

---
 media/libstagefright/MediaDefs.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/media/libstagefright/MediaDefs.cpp b/media/libstagefright/MediaDefs.cpp
index 2a50692..670cc3d 100644
--- a/media/libstagefright/MediaDefs.cpp
+++ b/media/libstagefright/MediaDefs.cpp
@@ -28,6 +28,10 @@ const char *MEDIA_MIMETYPE_VIDEO_MPEG4 = "video/mp4v-es";
 const char *MEDIA_MIMETYPE_VIDEO_H263 = "video/3gpp";
 const char *MEDIA_MIMETYPE_VIDEO_MPEG2 = "video/mpeg2";
 const char *MEDIA_MIMETYPE_VIDEO_RAW = "video/raw";
+const char *MEDIA_MIMETYPE_VIDEO_VC1 = "video/vc1";
+const char *MEDIA_MIMETYPE_VIDEO_RV = "video/rv";
+const char *MEDIA_MIMETYPE_VIDEO_FLV1 = "video/flv1";
+const char *MEDIA_MIMETYPE_AUDIO_RA = "video/ra";
 
 const char *MEDIA_MIMETYPE_AUDIO_AMR_NB = "audio/3gpp";
 const char *MEDIA_MIMETYPE_AUDIO_AMR_WB = "audio/amr-wb";
-- 
2.1.4

