From bbb743bc02edd04281a3d399cd9024151739f7e6 Mon Sep 17 00:00:00 2001
From: CTXz <patrick_pedersen_02032001@yahoo.com>
Date: Sat, 24 Oct 2015 18:53:20 +0200
Subject: [PATCH 1/1] FFmpegExtractor.cpp break patch

---
 extractor/FFmpegExtractor.cpp | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/extractor/FFmpegExtractor.cpp b/extractor/FFmpegExtractor.cpp
index 0b39032..982d3ac 100644
--- a/extractor/FFmpegExtractor.cpp
+++ b/extractor/FFmpegExtractor.cpp
@@ -2071,9 +2071,3 @@ MediaExtractor *CreateFFmpegExtractor(const sp<DataSource> &source, const char *
 }
 
 }  // namespace android
-
-extern "C" void getExtractorPlugin(android::MediaExtractor::Plugin *plugin)
-{
-    plugin->sniff = android::SniffFFMPEG;
-    plugin->create = android::CreateFFmpegExtractor;
-}
-- 
2.1.4

